exporter/datadogexporter/traces_exporter_windows.go

```diff
@@ -0,0 +1,38 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build windows
+
+package datadogexporter
+
+import (
+	"context"
+	"errors"
+
+	"go.opentelemetry.io/collector/consumer/pdata"
+	"go.uber.org/zap"
+)
+
+type traceExporterStub struct {}
+
+func newTraceExporter(logger *zap.Logger, cfg *Config) (*traceExporterStub, error) {
+	return &traceExporterStub{}, errors.New("datadog trace export is currently not supported on Windows")
```

Comment:
Why not just return `nil` instead of the stub?

Reply:
You would have to change where you define things a bit so consider this as solved.

Reply:
There were some compilation issues in `factory.go` because we call some methods on the `traceExporter` return value, such as [pushTraceData](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/1266/files#diff-c9e9f39ffda08a2af74c0d54d37f21e14cdb95c4136cdbe238de6be2bfe31389R133). So just returning stubs here made all those issues go away